protocol-configuration/permissions.ts

```diff
@@ -24,18 +24,27 @@ export const permissions = {
     'lusdPSMFeiSkimmer'
   ],
   GUARDIAN_ROLE: ['multisig', 'pcvGuardian', 'pcvSentinel'],
-  ORACLE_ADMIN_ROLE: ['collateralizationOracleGuardian', 'optimisticTimelock', 'opsOptimisticTimelock'],
+  ORACLE_ADMIN_ROLE: [
+    'collateralizationOracleGuardian',
+    'optimisticTimelock',
+    'opsOptimisticTimelock',
+    'tribalCouncilTimelock'
+  ],
   SWAP_ADMIN_ROLE: ['pcvEquityMinter', 'optimisticTimelock'],
   BALANCER_MANAGER_ADMIN_ROLE: [],
-  PSM_ADMIN_ROLE: [],
+  PSM_ADMIN_ROLE: ['tribalCouncilTimelock'],
   TRIBAL_CHIEF_ADMIN_ROLE: ['optimisticTimelock', 'tribalChiefSyncV2'],
+  FUSE_ADMIN: ['optimisticTimelock', 'tribalChiefSyncV2'],
   VOTIUM_ADMIN_ROLE: ['opsOptimisticTimelock', 'protocolPodTimelock'],
   PCV_GUARDIAN_ADMIN_ROLE: ['optimisticTimelock'],
   METAGOVERNANCE_VOTE_ADMIN: ['feiDAOTimelock', 'opsOptimisticTimelock'],
   METAGOVERNANCE_TOKEN_STAKING: ['feiDAOTimelock', 'opsOptimisticTimelock'],
   METAGOVERNANCE_GAUGE_ADMIN: ['feiDAOTimelock', 'optimisticTimelock'],
   ROLE_ADMIN: ['feiDAOTimelock', 'tribalCouncilTimelock'],
   POD_METADATA_REGISTER_ROLE: ['tribalCouncilSafe'],
+  FEI_MINT_ADMIN: ['core', 'feiDAOTimelock', 'roleBastion'],
```

Comment:
why is core a FEI_MINT_ADMIN

Reply:
Shouldn't be, have removed from this permissions mapping. Shouldn't be in there - will make sure the e2e tests don't expect core to be a FEI_MINT_ADMIN